lecture00-introduction/introduction.tex

```diff
@@ -122,10 +124,18 @@
 
 \end{frame}
 
-\begin{frame}{Some questions}
-
-see menti.com
-
+\begin{frame}{Survey regarding preknowledge}
+    \begin{center}
+        \textcolor{black}{
+            \qrcode[height=4.5cm]{https://www.menti.com/<snip>}
```

Comment:
I've checked how menti.com works, I think we can keep the QR code, the URL stays the same when entering the code twice.